packages/jsii/lib/util.ts

```diff
@@ -8,6 +8,9 @@ export function filterEmpty(_key: any, value: any): any {
     if (typeof value === 'object' && Object.keys(value).length === 0) {
         return undefined;
     }
+    if (!value) {
```

Comment:
are we sure we want to omit empty strings, `false` and `0`?

Reply:
So:
* There is no `number` field in the assembly specification, so there shouldn't be any `0`.
* All strings that can possibly be blank are also nullable (`namespace`, `Method#name` - since constructor are nullable).
* All `boolean` are also nullable. Also, `undefined == false`, so this one is really a no-brainer.